Mos/Localization/en.lproj/Localizable.strings

```diff
@@ -17,9 +17,9 @@
 "Quit" = "Quit";
 "Hide Icon" = "Hide Icon";
 
-// Notifaction
+// Notification
 "Mos is running in the background" = "Mos is running in the background";
-"Mos's status bar icon is hidden" = "Mos's status bar icon is hidden";
+"Mos' status bar icon is hidden" = "Mos' status bar icon is hidden";
```

Comment:
"Mos status bar icon is hidden" is enough.

Reply:
Either one can work. The one I used could be arguable more correct as it is possessive saying that it is the status bar icon of Mos.